Xamarin.Forms.Platform.Android/AppCompat/TabbedPageRenderer.cs

```diff
@@ -679,6 +679,8 @@ void UpdateTabIcons()
 				Page child = Element.Children[i];
 				TabLayout.Tab tab = tabs.GetTabAt(i);
 				SetTabIconImageSource(child, tab);
+				if (!string.IsNullOrEmpty(child.AutomationId))
+					tab.SetContentDescription(child.AutomationId);
```

Comment:
If we alter `SetContentDescription` here, we're actually changing the text that TalkBack will announce. So users with the screen reader on will now here "Auto 1" instead of "Page 1" as expected. Not good!